### PR TITLE
feat: client redirects

### LINKED ISSUE #985: Support Client Redirect to avoid broken links
Don't worry. Could you help list all broken links after this PR gets merged? And we could seek help from @SkyeYoung to implement this.

_Originally posted by @juzhiyuan in https://github.com/apache/apisix-website/pull/974#discussion_r836976994_

### PR DESCRIPTION
**Pull request by SkyeYoung**
Fixes: #985 

Changes:

- client redirects to some broken links

Screenshots of the change:

none

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -301,7 +301,22 @@ module.exports = {
...
+      {
+        redirects: [
+          {
+            from: ['/docs/general/community', '/docs/general/subscribe-guide'],
+            to: '/docs/general/join',
+          },
+          {
+            from: '/docs/general/submit-issue',
+            to: '/docs/general/contributor-guide',
+          },
```
Comment (pottekkat) [inline website/docusaurus.config.js]: We should not need this right?

**Review by pottekkat — COMMENTED** [review]
Tested the redirect and it works as expected: https://deploy-preview-986--apache-apisix.netlify.app/docs/general/community

File: website/docusaurus.config.js (hunk 1 above)
Comment (SkyeYoung) [reply website/docusaurus.config.js:316]: image

It's from here.

Comment (SkyeYoung) [reply website/docusaurus.config.js]: I found the problem. Thx.

Comment (SkyeYoung) [reply website/docusaurus.config.js]: I fixed this.

**Review by pottekkat — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Review by yzeng25 — APPROVED** [review]

**Pull request merged**